flytekit/bin/entrypoint.py

```diff
@@ -185,6 +180,17 @@ def _dispatch_execute(
         exit(1)
 
 
+def get_traceback_str(e: Exception) -> str:
```

Comment:
do we gain anything from having this function? Can you show a few examples of how this improves on `traceback.format_exc()`?